services/common/src/wrapper/mod.rs

```diff
@@ -0,0 +1,4 @@
+//! Pool Service backend for the message bus and gRPC
```

Comment:
This comment looks like it should be removed.

Reply:
Done